src/sentry/web/client_config.py

```diff
@@ -236,7 +231,17 @@ def get_client_config(request=None):
     }
     if user and user.is_authenticated:
         context.update(
-            {"isAuthenticated": True, "user": serialize(user, user, DetailedSelfUserSerializer())}
+            {
+                "isAuthenticated": True,
+                "user": user_service.serialize_users(
+                    [user.id],
+                    detailed=2,
+                    auth_context=AuthenticationContext(
+                        auth=getattr(request, "auth", None),
+                        user=request.user,
+                    ),
+                )[0],
```

Comment:
Could we extract this upward to
```python
(user,) = user_service.serialize_users(
    [user.id],
    ...
)
```
?

I never like seeing `[0]` in this position because it's not immediately obvious whether we need to guard against a length != 1, or if we always want the first of multiple values, or whatever.